app/controllers/comments_controller.rb

```diff
@@ -65,6 +66,6 @@ def set_comment
 
     # Only allow a list of trusted parameters through.
     def comment_params
-      params.require(:comment).permit(:author_id, :photo_id, :body_text)
+      params.require(:comment).permit(:author_id, :photo_id, :body, )
```

Comment:
Don't know if the trailing comma at the end of body will break anything but just double check and make sure it doesn't.